### ISSUE TITLE
Show challenge owner directly on challenge overview page

### ISSUE THREAD

**Issue by sun-geo** [maproulette/maproulette3#1577]
More transparency to challenge information. It seems the challenge owner information isn't very easy to find, i think, as it could be. Currently the (only?) way is (deep in the system) while working on a specific challenge task item under [1] (tripple dot menu/Contact Challenge Owner/Message OSM-USERNAME through OSM)
Please add the owner information preferable also to the challenge overview page [2]
  
[1]
![image](https://user-images.githubusercontent.com/47290585/108905594-30853780-7620-11eb-92a4-ed313d91ee68.png)
<sup>https://maproulette.org/challenge/14261/task/63107121</sup>

[2]
![image](https://user-images.githubusercontent.com/47290585/108903861-164a5a00-761e-11eb-9624-318fb61080c5.png)
<sup>https://maproulette.org/browse/challenges/14261</sup>

**Comment by mvexel** [comment]
Agreed adding challenge owner name to the main challenge page would be good. Thanks for the suggestion.

**Comment by jlewin** [comment]
This appears to have been fixed and seemingly can be closed:

![Image](https://github.com/user-attachments/assets/719a67b7-c44c-4359-9ae5-c7fd2d760fc3)

**Comment by CollinBeczak** [comment]
Nice catch! resolved in https://github.com/maproulette/maproulette3/pull/2429
